src/Cedar/Proto_PPP.c

```diff
@@ -2322,8 +2323,7 @@ bool PPPRejectLCPOptionsEx(PPP_SESSION *p, PPP_PACKET *pp, bool simulate)
 		return false;
 	}
 
-	PPPSendPacketAndFree(p, ret);
-	return true;
+	return PPPSendPacketAndFree(p, ret);

```

Comment:
return true implies that the packet should be rejected. I would leave it.

Reply:
You're actually right, that indeed means the packet was processed, and if the pipe is broken that doesn't change the fact. Probably the reason I did it in the first place.